src/api/Feed.js

```diff
@@ -1201,16 +1207,18 @@ class Feed extends Base {
 
         const cachedItems = this.getCachedItems(this.id);
         if (cachedItems) {
-            const updatedFeedItems = cachedItems.items.map((item: Comment | Task | TaskNew | BoxItemVersion) => {
-                if (item.id === id) {
-                    return {
-                        ...item,
-                        ...updates,
-                    };
-                }
+            const updatedFeedItems = cachedItems.items.map(
+                (item: Comment | Task | TaskNew | BoxItemVersion | AppActivityItem) => {
```

Comment:
It looks like this is duplicated in the `FeedItems` type declaration. Can it be shared via a `FeedItem` type?

Reply:
Sounds great!